crates/jstzd/src/task/jstzd.rs

```diff
@@ -148,6 +178,18 @@ impl Jstzd {
         }
         Ok(())
     }
+
+    /// Wait for the baker to bake at least `level` blocks.
+    async fn wait_for_block_level(node_endpoint: &Endpoint, level: i64) -> Result<()> {
+        let ready = retry(10, 1000, || async {
```

Comment:
Note: Would be nice if retry returned a `Result<()>` instead. No action required in this PR.